_data/blogs.yml

```diff
@@ -7,7 +7,7 @@
   title: Interview with a teacher
   img: samyakbvs.jpeg
   author_name: Samyak Jain
-
+  
```

Comment:
Remove this spacing

Reply:
![image](https://user-images.githubusercontent.com/34100642/34349435-a426196e-ea43-11e7-8408-8b51d65c54a1.png)
I added the line because it was too close to blog number 6, so I added it to be separate

Reply:
@anshumanv

Reply:
Extra line is to be added, but please remove the space on the line you added as seen in diff.